docs: suggestions for updates to cluster docs (#4253)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/clusters/creating-konvoy-cluster-advanced/index.md

### Use the upload YAML form

1.  In the selected workspace Dashboard, select the **Add Cluster** option in the **Actions** dropdown menu at the top right.

1.  On the Add Cluster page, select **Upload YAML to Create a Cluster** and provide advanced cluster details:

    - **Workspace**: The workspace where this cluster belongs (if within the Global workspace).
    - **Cluster YAML**: Paste or upload your customized set of cluster objects into this field. Only valid YAML will be accepted.
    - **Add Labels**: By default, your cluster has labels that reflect the infrastructure provider provisioning. For example, your AWS cluster may have a label for the data center region and `provider: aws`. Cluster labels are matched to the selectors created for [projects][projects]. Changing a cluster label may add or remove the cluster from projects.

> 1. Click **Create** to begin provisioning the Konvoy cluster. This step may take a few minutes, taking time for the cluster to be ready and fully deploy its components. The cluster will retry to join automatically and should resolve once it is fully provisioned.

Reviewer: ```suggestion
1. Click **Create** to begin provisioning the Konvoy cluster. This step may take a few minutes, taking time for the cluster to be ready and fully deploy its components. The cluster automatically tries to join, and should resolve after it is fully-provisioned.
```
